test/unit/ios_app_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class IosAppTest < Minitest::Test
+  let(:browser) { Browser.new(ua: $ua['IOS_APP']) }
+
+  test 'detect as ios' do
+    assert browser.ios?
+  end
+
+  test 'detect as safari' do
+    assert browser.safari?
+  end
+
+  test 'detect as app' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.